node/package.json

```diff
@@ -41,6 +41,7 @@
     "debug-logtron": "4.0.0",
     "duplexer": "^0.1.1",
     "faucet": "0.0.1",
+    "format-stack": "4.0.0",
```

Comment:
this is for `itape` support. I can remove this if wanted.

itape supports `itape test/index.js --format-stack` where it runs your tests with the local `format-stack` library imported. This allows you to not edit the tests to import format-stack for debugging purposes.

itape does require a local copy of the library though because you cant really require global npm modules because of how nodes module system works.

This means `format-stack` is a weird transitive dependency thing.